main.py

```diff
@@ -0,0 +1,142 @@
+from PyQt4 import QtGui
```

Comment:
After this, another commit was included (for Windows functionality) - [this.](https://github.com/Parth-Vader/FB-Spider/commit/d9a373712be1a0f492c53a153c59e9ac83b886d8)
Make the suitable changes here too.